airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py

```diff
@@ -37,7 +38,7 @@ class DefaultFileBasedStream(AbstractFileBasedStream, IncrementalMixin):
     ab_file_name_col = "_ab_source_file_url"
     airbyte_columns = [ab_last_mod_col, ab_file_name_col]
 
-    def __init__(self, cursor: FileBasedCursor, **kwargs):
+    def __init__(self, cursor: FileBasedCursor, **kwargs):  # type: ignore
```

Comment:
`Dict[str, Any]`?

Reply:
`Dict` isn't the right type because we're passing the objects directly.

[According to the mypy cheatsheet, we only need to specify the type of the values since mypy assumes `kwargs` is a dictionary](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html).

Updates to `Any`

Reply:
Cool, thanks for pointing that out.